pkg/labels/labels.go

```diff
@@ -0,0 +1,10 @@
+package labels
+
+const (
+	// WorkerRole is the role label of worker nodes
+	WorkerRole = "node-role.kubernetes.io/worker"
+	// MasterRole is the old role label of control plane nodes
+	MasterRole = "node-role.kubernetes.io/master"
+	// ControlPlaneRole is the new role label of control plane nodes
+	ControlPlaneRole = "node-role.kubernetes.io/control-plane"
+)
```

Comment:
what about the IsControlPlane func?

Reply:
Sure, I thought you meant only copying the  labels from the files, but I don't mind adding `IsControlPlane` as well - but I think I'll put in a different pkg (labels doesn't seem like a great fit)